cocotb/share/makefiles/Makefile.sim

```diff
@@ -30,8 +30,71 @@
 # This is a simple wrapper Makefile to pull in the appropriate Makefile for
 # the desired simulator and set up common paths for the sims to use to build with
 
+.PHONY: all
```

Comment:
That's not really related to this PR, it should be a separate commit at least.

Reply:
Done.